packages/hub/src/session.ts

```diff
@@ -21,9 +21,9 @@ export class Session implements SessionInterface {
   public release?: string;
   public sid: string = uuid4();
   public did?: string;
-  public timestamp: number = Date.now();
-  public started: number = Date.now();
-  public duration: number = 0;
+  public timestamp: number = timestampInSeconds;
+  public started: number = timestampInSeconds;
```

Comment:
⚠️  `timestampInSeconds` is a function (isn't it? And why doesn't TypeScript help here...)

BTW -- what's the best way we could initialize both `timestamp` and `started` to the exact same value?

Reply:
>  timestampInSeconds is a function (isn't it? And why doesn't TypeScript help here...)

Yes, I fixed the type

> BTW -- what's the best way we could initialize both timestamp and started to the exact same value?

Updated the `Session` class to use the same value for both `timestamp` and `started` in class constructor.